velox/exec/tests/TableEvolutionFuzzerTest.cpp

```diff
@@ -0,0 +1,650 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/connectors/hive/HiveConnector.h"
+#include "velox/connectors/hive/HiveConnectorSplit.h"
+#include "velox/dwio/dwrf/RegisterDwrfReader.h"
+#include "velox/dwio/dwrf/RegisterDwrfWriter.h"
+#include "velox/exec/Cursor.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/exec/tests/utils/TempDirectoryPath.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
+
+#include <folly/init/Init.h>
+#include <gflags/gflags.h>
+#include <gtest/gtest.h>
+
+#include <filesystem>
+
+DEFINE_uint32(seed, 0, "");
+DEFINE_int32(duration_sec, 30, "");
+DEFINE_int32(column_count, 5, "");
+DEFINE_int32(evolution_count, 5, "");
+
+namespace facebook::velox::exec::test {
+
+namespace {
+
+constexpr int kVectorSize = 101;
+
+VectorFuzzer::Options makeVectorFuzzerOptions() {
+  VectorFuzzer::Options options;
+  options.vectorSize = kVectorSize;
```

Comment:
Because these encodings don't affect the schema of the table, and the vector will be write to file immediately, and these files also don't preserve these encodings, we will lose them when we read the files back, so there is no need to generate them in the first place.